### DISCUSSION TITLE
Customise default ChangePassword page

### DISCUSSION THREAD

**Discussion by robmuk0** [OrchardCMS/OrchardCore#11490] [Q&A]
Hi, 

I'd like to customise a couple of pages including ChangePassword. Could someone please point me in the right direction as to how that can be achieved?

I have tried creating a ChangePassword.cshtml view in my solution, which didn't work.

Thanks

**Comment by lampersky** [comment]
Create your custom theme, add a few view overrides (ForgotPassword, ForgotPasswordConfirmation, ResetPassword, ResetPasswordConfirmation etc.) 

![image](https://user-images.githubusercontent.com/6403130/161594353-8b8039b6-3989-4f7a-a480-c47df5796672.png)

run the project, select custom theme as current one and go to few **Security > Settings** sections, and select same settings as shown below: 

**/Admin/Settings/userLogin**
![image](https://user-images.githubusercontent.com/6403130/161593435-34fd26c4-641c-485b-b044-dfa8c83240ac.png)


**/Admin/Settings/userResetPassword**
![image](https://user-images.githubusercontent.com/6403130/161593486-d86b4e1a-1262-472c-9500-dfedb2cb3c51.png)


**Admin/Settings/userRegistration**
![image](https://user-images.githubusercontent.com/6403130/161593613-4eb026f6-7b90-455f-9df3-b2fdaaffb590.png)

**Comment by robmuk0** [comment]
Hi, thanks for the detailed guide. I am trying to add the top navbar and the side menu to change password page, which means just using the the standard layout instead of login layout. That's just needed for the change password page and all other pages can remain as they are. 

Your solution probably requires me to override ALL of the pages you've shown in the screenshot I believe. Because when I select the site theme for login page, all of the user related pages start using that.

**Reply by ns8482e** [reply]
All these pages rendered using a theme - you can select either to use site theme or admin theme. by default these pages uses admin theme.   As shown above, it provides different settings for different features, you may choose to "check - use site theme" for all features as per above or you can just enable it for one feature.
